docs/ContribOperators.md

```diff
@@ -2906,6 +2907,55 @@ This version of the operator has been available since version 1 of the 'com.micr
 </dl>
 
 
+### <a name="com.microsoft.MoEBlock"></a><a name="com.microsoft.moeblock">**com.microsoft.MoEBlock**</a>
```

Comment:
why not name the operator as MoE instead of MoEBlock

Reply:
I don't have strong preference: I can rename it if MoE is preferred

Reply:
MoE has many types. It is better to be more specified in naming. Could we add some description about the background (links to paper or model) that using such  MoE?

For example, GPT style MOE model: https://www.modelscope.cn/models/PAI/nlp_gpt3_text-generation_0.35B_MoE-64/summary. It is based on switch transformer (https://arxiv.org/pdf/2101.03961.pdf). A proper naming for that layer could be SwitchFFN.